README.adoc

```diff
@@ -34,30 +34,15 @@ Run the following command to find out your Go version.
 $ go version
 ----
 
-*You must at least have Go version 1.8.*
+*You must at least have Go version 1.9.*
```

Comment:
Do we really need to have this? I think we can stick with 1.8 for now because we don't compile `deb`. At least that is what you told me ;)

Reply:
yes, right, let me fix that